python/cudf/cudf/_lib/rolling.pyx

```diff
@@ -19,7 +19,8 @@ cimport cudf._lib.includes.rolling as cpp_rolling
 
 
 def rolling(inp, window, min_periods, center, op):
-    from cudf.core.column import Column
+    from cudf.core.buffer import Buffer
+    from cudf.core.column import build_column
```

Comment:
Are we able to move the `import`s out of this function or do they need to be here for some reason?

Reply:
Hmm, I'm worried about introducing circular import issues which sometimes don't show up locally but do so on CI. Seems like it shouldn't be a problem for `rolling.pyx` but it's difficult to say for the other Cython modules.

Happy to try moving these up to the module level if you feel strongly about it

Reply:
That's what I was wondering. Maybe we can revisit later then.

Reply:
Cool - marking resolved! Thanks!